config/samples/advertise_two_neighs_sameconfig.yaml

```diff
@@ -0,0 +1,43 @@
+kind: FRRConfiguration
+metadata:
+  name: test
+  namespace: frr-k8s-system
+spec:
+  bgp:
+    routers:
+    - asn: 64512
+      neighbors:
+      - address: 172.30.0.3
+        asn: 4200000000
+        ebgpMultiHop: true
+        holdTime: 0s
+        keepaliveTime: 0s
+        password:
```

Comment:
wdyt about adding the secret as well?

Reply:
made things even simpler, removed passwords from all and adding a _with_password custom example